src/test/java/org/magicdgs/readtools/TestResourcesUtils.java

```diff
@@ -36,17 +36,49 @@
     // current directory for the tests
     private static final String CURRENT_DIRECTORY = System.getProperty("user.dir");
 
-    /** The root file directory for main resources files.*/
+    /** The root file directory for main resources files. */
     public static final String READTOOLS_MAIN_RESOURCES_DIRECTORY =
             new File(CURRENT_DIRECTORY, "src/main/resources").getAbsolutePath() + "/";
 
     /** The root file directory for test resource files. */
     public static final String READTOOLS_TEST_ROOT_FILE_DIRECTORY =
             new File(CURRENT_DIRECTORY, "src/test/resources").getAbsolutePath() + "/";
 
-    /** The directory for example data. */
-    public static final String READTOOLS_EXAMPLE_DATA_DIRECTORY =
-            new File(CURRENT_DIRECTORY, "/testdata").getAbsolutePath() + "/";
+    /**
+     * Directory for example data used in the Walkthrough. This data belongs to the documentation
```

Comment:
All this should go to the developmental wiki (#234). Should open a PR for remember to add!